lib/Runtime/Language/InlineCache.h

```diff
@@ -964,6 +964,72 @@ namespace Js
         void Unregister(ScriptContext * scriptContext);
     };
 
+    // Two-entry Type-indexed circular cache
```

Comment:
Yes. Tried 1-, 2-, and 3-entry caches. From 1 to 2 saw significant perf gain; from 2 to 3 no gain. Therefore chose 2-entry for simplicity.

---
In reply to: [68804491](https://github.com/Microsoft/ChakraCore/pull/1198#discussion_r68804491) [](ancestors = 68804491)